ex1/exceptions.py

```diff
@@ -37,6 +37,32 @@
 # Use the function search_n from the functions module inside a new function
 # also named search_n. The function should do the same as functions.search_n
 # but if the variable is not found in the list then raise a ValueError.
+def search_n(List, x):
+	"""
+	Parameters
+	----------
+	List : list of optional
+	x    : optional
+
+	Returns
+	-------
+	int
+		Index of the found element
+	optional
+		Value of the found element
+
+	Raise
+	------
+	ValueError
+		if no element found
+    	"""
+	c = 0 # index counter
```

Comment:
You should have used the search_n function from `functions.py`. Mainly to practice importing and working with multiple functions that have the same name.